config/locales/metadata/metadata-es.yml

```diff
@@ -17,4 +17,4 @@ es:
       template_file_anchor: este archivo de plantilla
       upload: Subir
       upload_metadata: Subir metadatos
-      upload_metadata_description: Para actualizar los metadatos de varias obras dentro de esta colección, cargue una hoja de cálculo con una fila por obra. Descargue %{link} para comenzar.
+      upload_metadata_description: Para actualizar los metadatos de varias obras dentro de esta colección, cargue una hoja de cálculo con una fila por obra. <br> Las columnas con <i>estrellas</i> no se pueden modificar. Ignore o elimine cualquier columna que no esté cambiando. <br> Descargue %{link} para comenzar.
```

Comment:
This is fine: "Los ingenieros del software y los científicos lo denominan a veces como «estrella»", according to Wikipedia.

Reply:
Sorry, can you please clarify what we want here instead?

Reply:
Sorry -- I meant that _estrella_ was fine in a technical context.  (I'd been trying to make sure we didn't need to translate is as _asterisco_.)